docs/jwst/associations/level2_asn_technical.rst

```diff
@@ -122,36 +124,37 @@ members *required*
   This is a list of the exposures to be used by the stage 2 processing
   tasks. This keyword is explained in detail in the next section.
 
+.. _asn-level2-members:
+
 ``members`` Keyword
 ^^^^^^^^^^^^^^^^^^^
 
-``members`` is a list of objects, each consisting of the following
-keywords
+``members`` is a list of dictionaries, one for each member exposure in the
+association. Each member has the following keywords.
 
 expname *required*
   The exposure file name
 
 exptype *required*
-  Type of information represented by the exposure. Possible values are
+  Type of information represented by the exposure. Possible
+  values are as follows. *Note that this is not the same as the exposure
+  ``EXP_TYPE`` header keyword.*
 
-  * ``science`` *required*
-    
-    Primary science exposure. For each product, only one exposure can
+  * ``science``: Primary science exposure. For each product, only one exposure can
     be ``science``.
     
-  * ``background`` *optional*
-    
-    Off-target background exposure to subtract.
+  * ``background``: Off-target background exposure to subtract.
```

Comment:
The use `off-target` might be misleading here, because it would be easy to interpret it to mean only exposures of a dedicated background target. But it can also be an exposure of the same target that has the target nodded to a different location in the FoV. So perhaps make it more generic, like just "Exposure to use for background subtraction" or "Background exposure to subtract" (which has the same syntax as the comment below for imprint).

Reply:
✅